docs/performance.md

```diff
@@ -1,112 +1,131 @@
 
-#### Managing Underperformance
+At Koor Technologies, Inc., managing performance has three primary components:
 
-Managing and identifying underperformance can be one of the most difficult responsibilities of a manager, but it's also one of the most important. As a people leader, to be fair and transparent with the team member, you want to address underperformance in a timely and structured way.
+- Team-level Objectives and Key Results (OKRs) -- stretch goals that align our efforts and drive company growth and development of new capabilities
+- Team-level Key Performance Indicators (KPIs) -- measures of operational health
+- Individual performance feedback -- information we can use to grow as individuals and to develop our careers
 
-Koor Technologies, Inc. will leverage GitLab's [skill-based and will-based performance model](https://about.gitlab.com/handbook/leadership/underperformance/#skills-based-performance) to identify the root cause of the team members' underperformance and with that determine the suitable remediation plan. The remediation plan will differ based on the type of performance issue.
+Our process is a light-weight adaptation of GitLab's process. You can learn more about their process through their pages on [OKRs](https://about.gitlab.com/company/okrs/), [KPIs](https://about.gitlab.com/handbook/ceo/kpis/), and [360 feedback](https://about.gitlab.com/handbook/people-group/360-feedback/).
 
-#### Remediation for underperformance
+#### Key aspects of Koor Technologies, Inc. performance management
 
-Managers should address performance concerns early and not wait until the performance problems have become unrecoverable. Feedback should be given early, it should be specific and documented. Remediation options will vary based on the circumstances, but as a best practice managers should leverage one or more of these approaches:
+The most important aspects of our performance framework are:
 
-1. Coaching - engage with the team member to help diagnose problem(s) and solution(s), perhaps with some outside guidance such as articles or courses. Requires written followup but not cc to peopleops.
-2. Written warning - an explicit notice that performance is not meeting the company's needs, citing examples of problems and clarity on expectations moving forward. (manager must cc [peopleops@koor.tech](mailto:peopleops@koor.tech))
-3. Performance Improvement Plan (PIP) - a formal plan documenting problems, expectations, and a timeline for resolution to avoid termination. (manager must cc [peopleops@koor.tech](mailto:peopleops@koor.tech)).
+- We have one north star \-- population growth. This helps us measure the results of all of our other efforts and prioritize what matters. At the end of the day, our contribution to the world will scale with the number of people living in cities powered by Koor Technologies, Inc.
```

Comment:
I almost did a little edit when I saw this to make it more literal, that we're after user population growth. But I like the metaphor, and in a way it could be a grand literal future.

Reply:
Also, as a matter of priority, I'm focused on establishing perf reviews for us, so kind of left the other parts intact, other than re-organizing the blocks.